src/Compilers/CSharp/Test/Emit/CodeGen/CodeGenNullCoalescingAssignmentTests.cs

```diff
@@ -51,15 +51,30 @@ static string GetString()
 }
 ";
 
-            var verifier = CompileAndVerify(source, expectedOutput: @"
+            CompileAndVerify(@"
+using System;
+class C
+{
+    static void TestNullable()
+    {
+        int? i1 = null;
+        Console.WriteLine(i1 ??= GetInt());
+    }
+    static int GetInt()
+    {
+        Console.WriteLine(""In GetInt"");
+        return 0;
+    }
+}
+").VerifyIL("C.TestNullable()", "");
+            /*var verifier = CompileAndVerify(source, expectedOutput: @"
 In GetInt
 0
 In GetString
 Test
 As Statement
 ");
 
-            // PROTOTYPE(null-operator-enhancements): lines 8 and 9 appear to be entirely redundant.
```

Comment:
This is fallout from https://github.com/dotnet/roslyn/blob/master/src/Compilers/CSharp/Portable/Lowering/LocalRewriter/LocalRewriter_NullCoalescingOperator.cs#L124, which unconditionally stores the lhs of a `x ?? y` expression, even if the lhs is a local and does not need to be stored.